medcat/utils/meta_cat/data_utils.py

```diff
@@ -210,6 +210,8 @@ def encode_category_values(data: Dict, existing_category_value2id: Optional[Dict
     for i in range(len(data)):
         if data[i][2] in category_value2id.values():
             label_data_[data[i][2]] = label_data_[data[i][2]] + 1
+
+    logger.info(f"Original label_data: {label_data_}")
```

Comment:
We want to use lazy formatting for logging. So instead of an f-string, use `%s` formatting.

Reply:
Oh yes I'll fix that!
Regarding the ordering of unpacking, I was going to only change the order for link 354, however Tom pointed out that it would make the most sense to have it `full_data` and then `undersampled_data`
That's why the long route

Reply:
Should I make the change for all logging statements to use lazy logging?

Reply:
Generally don't want to change the purpose of a PR.
But I don't think there's too many. So why not.

As for unpacking - yes, I agree. This order makes more sense in general. Just thought I'd mention it.